404.html

```diff
@@ -4,6 +4,10 @@
 <head>
 	<link rel="stylesheet" href="https://xorol.github.io/style.css"></a>
 	<title>Uh oh! 404</title>
+	<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+	<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+	<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+	<link rel="manifest" href="/site.webmanifest">
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consistency in referencing external resources

Consider using a consistent base URL for external resources across all HTML files to ensure uniformity and reduce potential errors in resource loading.

```suggestion
	<link rel="apple-touch-icon" sizes="180x180" href="https://xorol.github.io/apple-touch-icon.png">
	<link rel="icon" type="image/png" sizes="32x32" href="https://xorol.github.io/favicon-32x32.png">
	<link rel="icon" type="image/png" sizes="16x16" href="https://xorol.github.io/favicon-16x16.png">
	<link rel="manifest" href="https://xorol.github.io/site.webmanifest">
```